web/pdf_print_service.js

```diff
@@ -157,8 +163,12 @@ PDFPrintService.prototype = {
         this._printResolution,
         this._optionalContentConfigPromise
       )
-        .then(this.useRenderedPage.bind(this))
-        .then(function () {
+        .then(() => {
+          if (!this.pdfDocument.isPureXfa) {
```

Comment:
Is this check still relevant, since I don't believe that we can get here now given the check at the start of this method?